src/activities/Elsa.Activities.Dropbox/Handlers/DropboxApiHandler.cs

```diff
@@ -1,18 +0,0 @@
-using System.Net.Http;
```

Comment:
I'm not sure I can delete it since it's public, but it looks like it's not used to have no value

Reply:
I don't think you can delete that namespace import, since `HttpRequestMessage` comes from that namespace.

Reply:
My comment was related to the entire file :)

Reply:
Ah. Yes, this entire file can be removed (and `ServiceCollectionExtensions` needs to be updated accordingly).